cvat/nginx.conf

```diff
@@ -47,6 +47,12 @@ http {
     client_max_body_size 1G;
 
     add_header X-Frame-Options deny;
+    add_header Referrer-Policy "strict-origin-when-cross-origin";
+    add_header X-Content-Type-Options "nosniff";
+    add_header X-Frame-Options "deny";
+    add_header Referrer-Policy "strict-origin-when-cross-origin";
+    add_header X-Content-Type-Options "nosniff";
+    add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; img-src 'self' data: https://app.cvat.ai; font-src 'self' https://fonts.gstatic.com; frame-src 'none'; object-src 'none'; base-uri 'self'; form-action 'self'; frame-ancestors 'none'; manifest-src 'self'; worker-src 'self'; 
```

Comment:
This config will likely require a different CSP, because this instance of NGINX hosts the backend, so it should allow resources referenced by pages generated by backend. In particular, you should examine the external resources loaded by pages like:

* `https://<domain>/api/docs/`
* `https://<domain>/api/swagger/`
* `https://<domain>/api/about/`
* `https://<domain>/django-rq/`
* `https://<domain>/admin/`